### PR TITLE
Feature/#342 multiple polls at a time

### LINKED ISSUE #342: As a lecturer I can have multiple polls running at the same time
AC:
- Multuple polls can be active at the same time
- Students can answer all active polls
- I can interact with all active polls like I can currently interact with the single active poll
- Students can interact with all active polls like they can currently interact with the single active poll

### PR DESCRIPTION
**Pull request by Xantocx**
# Resolves #342

- Multuple polls can be active at the same time
- Students can answer all active polls
- I can interact with all active polls like I can currently interact with the single active poll
- Students can interact with all active polls like they can currently interact with the single active poll

NOTE: This is a revert of the "feature" of only have one running poll at a time. The original feature still exists, but it's not applied.

### CODE REVIEW

File: app/models/poll.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Poll < ApplicationRecord
   validates :title, presence: true
   validates :is_multiselect, inclusion: { in: [true, false] }
   validates :is_active, inclusion: { in: [true, false] }
-  validate :only_one_poll_active
+  #validate :only_one_poll_active
```
Comment (JoeAtHPI) [inline app/models/poll.rb]: why comment out and not delete? besides I am missing the test to ensure the wanted behavior

Comment (Xantocx) [reply app/models/poll.rb]: This line could be removed, but I am not sure about the method executed there. What do you think?
Test will be added.

Comment (JoeAtHPI) [reply app/models/poll.rb]: remove them both

Comment (Xantocx) [reply app/models/poll.rb]: Done.

**Pull request merged**